### ISSUE TITLE
expected error

### ISSUE THREAD

**Issue by ppxasjsm** [michellab/Sire#196]
Hi, I get a segmentation fault when running tests on feat_nb branch, is this a to be expected failure?

Here is the error:
```
33275
Thrown from FILE: /Users/toni_brain/Projects/git/Sire/corelib/src/libs/SireIO/amberprm.cpp, LINE: 3512, FUNCTION: auto SireIO::AmberPrm::assertSane()::(anonymous class)::operator()() const" 
E......E..FSegmentation fault: 11
```

**Comment by chryswoods** [comment]
Yes, I was seeing that too today. Annoying to debug as it disappeared when I ran it under gdb. I am going to fix it once everything is ready with the workshop image

**Comment by chryswoods** [comment]
Been investigating further. The AmberPrm::assertSane exception is correct behaviour for your version and has been fixed in devel. Use a pull request or otherwise merge in the changes from devel to feat_nb to bring in the fixes for this.

The segfault is unrelated to the assertSane exception and is the segfault I was talking about in the last comment. It only appears on Linux and, I think, is a threading issue somewhere in the AmberPrm code. I am working on fixing it this morning.

**Comment by chryswoods** [comment]
Both of these errors (assertSane and segfault) are now fixed in devel. These are in the sire_2018_1_0_pre2.run file you can download and test (if you want - see 2018.1 issue #195)
